src/ts/components/seekbar.ts

```diff
@@ -702,9 +702,10 @@ export class SeekBar extends Component<SeekBarConfig> {
     this.setPosition(this.seekBarPlaybackPosition, percent);
 
     // Set position of the marker
-    let px = (this.config.vertical ? this.seekBar.height() : this.seekBar.width()) / 100 * percent;
+    let totalSize = (this.config.vertical ? (this.seekBar.height() - this.seekBarPlaybackPositionMarker.height()) : this.seekBar.width());
+    let px = (totalSize) / 100 * percent;
     if (this.config.vertical) {
-      px = this.seekBar.height() - px;
+      px = this.seekBar.height() - px - this.seekBarPlaybackPositionMarker.height();
```

Comment:
Taking the height into account here introduces another issue.... the position marker is not centered around the mouse cursor any longer. Noticeable when you click at the top or bottom. You'll see that the center position of the marker and the cursor do not match.

Reply:
It's not ideal but let's leave it like this. It's definitely an improvement to the issues we had before 👍

Reply:
Yes I guess it aligns just below the mouse.